apps/nextjs-website/react-components/components/Cards/Cards.tsx

```diff
@@ -7,68 +7,79 @@ import { CtaButtons } from '../common/Common';
 import { Title, Subtitle, Body } from '../common/Common';
 import { CardsItemContainer } from './Cards.helpers';
 
-const Cards = ({ items, theme, text, ctaButtons }: CardsProps) => {
-  const background = theme === 'dark' ? 'primary.dark' : 'background.paper';
+const Cards = ({
+  items,
+  theme,
+  text,
+  ctaButtons,
+  textPosition = 'left',
+}: CardsProps) => {
+  const { palette } = useTheme();
+  const background =
+    theme === 'dark' ? palette.custom.backgroundColorDark : 'background.paper';
```

Comment:
Make it consistent.

Either use 'custom.backgroundColorDark' if valid or palette.background.paper